src/gui/guiFormSpecMenu.cpp

```diff
@@ -2162,11 +2162,11 @@ void GUIFormSpecMenu::parseBox(parserData* data, const std::string &element)
 			colors = {tmp_color, tmp_color, tmp_color, tmp_color};
 		} else {
 			colors = style.getColorArray(StyleSpec::COLORS, {0x0, 0x0, 0x0, 0x0});
+			std::array<video::SColor, 4> bordercolors = style.getColorArray(
+				StyleSpec::BORDERCOLORS, {0x0, 0x0, 0x0, 0x0});
+			std::array<s32, 4> borderwidths = style.getIntArray(
+				StyleSpec::BORDERWIDTHS, {0, 0, 0, 0});
```

Comment:
Mistake? `bordercolors`  and `borderwidths` are used outside this scope,

Reply:
Yup, mistake.